api/models/risk_of_bias_criteria.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const bookshelf = require('../../config').bookshelf;
+const BaseModel = require('./base');
+require('./risk_of_bias');
```

Comment:
To keep it consistent with the rest of the code, move the require('./MODEL') to before the other requires (see https://github.com/opentrials/api/blob/master/api/models/trial.js for an example of what I mean)